roles/kubectl-proxy-systemd/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+# Creates a systemd unit and starts kubectl proxy.
+# https://kubernetes.io/docs/user-guide/kubectl/v1.8/#proxy
+
+- name: Install jq binary
+  get_url:
+    url: https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64
+    dest: /usr/bin/jq
+    mode: 0755
+
+- name: Template kubectl proxy systemd unit
+  template:
+    src: kubectl-proxy.service.j2
+    dest: /etc/systemd/system/kubectl-proxy.service
+  register: template_kubectlproxy_unit
+
+- name: Systemctl daemon-reload when appropriate
+  shell: >
```

Comment:
does the systemd module do this function now?

Reply:
indeed it does, thanks.